Resources/Prototypes/Entities/Mobs/Player/dragon.yml

```
@@ -127,14 +127,16 @@
     interactFailureSound:
       path: /Audio/Animals/space_dragon_roar.ogg
     soundPerceivedByOthers: false # A 75% chance for a loud roar would get old fast.
+  - type: Damageable # DeltaV - Add Damageable component to add a damageModifierSet
   - type: MeleeWeapon
```

Seems like leftover from previous commits?
```suggestion
    soundPerceivedByOthers: false # A 75% chance for a loud roar would get old fast.
  - type: MeleeWeapon
```

---

I didn't catch that, your suggestion is merged. Thanks